ansible/roles/ooni-backend/vars/main.yml

```diff
@@ -3,13 +3,21 @@ enable_dns_helper: yes
 enable_dns_discovery_helper: yes
 enable_http_return_json_headers_helper: yes
 enable_tcp_echo_helper: yes
-enable_ssl_helper: no
 enable_daphn3_helper: yes
-enable_tls_bouncer: no
-enable_tls_collector: no
-enable_tls_web_connectivity: no
 
 dns_resolver_address: "8.8.8.8:53"
+dns_test_helper_address: "213.138.109.232:57004"
+http_return_json_headers_address: "http://93.95.227.200"
```

Comment:
OK let's deploy some then, now that we have the alternate section can we also add multiple test-helpers in case some cannot be accessed or are blocked from the client side?